support-frontend/assets/components/svgs/securePadlock.tsx

```diff
@@ -0,0 +1,31 @@
+type Props = {
+	colour: string;
+	opacity?: number;
+};
+
+export default function SecurePadlock({
+	colour,
+	opacity = 1,
```

Comment:
These are now passed as props. This feels more intuitive than controlling via css with `fill: currentColour` (which wasn't working correctly with the additional div). I'm interested in thoughts on this approach!